client/src/components/layout/ReviewsSection.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+import ReviewTile from "./ReviewTile"
+
+const ReviewsSection = (props) => {
+    
+    const [reviews, setReviews] = useState([])
+    
+    const getReviews = async () => {
```

Comment:
this is requiring 2 separate fetch requests on the same page - the destination show page. Instead, when we fetch for a destination, we should also query for the associated reviews in the back end, and set those reviews in state up in the `DestinationShow` component